src/Entity/FileEntity.php

```diff
@@ -237,7 +240,7 @@ public static function preDelete(EntityStorageInterface $storage, array $entitie
    */
   public function updateBundle($type = NULL) {
     if (!$type) {
-      $type = file_get_type($this);
+      $type = $this->getType();
```

Comment:
If you look at the description of the old method, it doesn't *get* the type, it determines it. And it is likely an internal helper function. lets try protected function determineType()